hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Hexagon game based  for haiku"
```

Comment:
I Built it but without dependence the package built but if I build it with dependence error comes says "libperl.so" is missing. this error come to a lot of recipes
And how to find source files like "cp Soucers\hexvexed $appDir" here how to find where the app built (how did they found the the app is on Soucers file )